Makefile

```diff
@@ -41,11 +41,48 @@ license-check: $(VENV) deps
 .PHONY: lint
 lint: $(VENV) deps
 	@echo "LINTING"
-	$(PYTHON) -m flake8 tests detection_rules --ignore D203 --max-line-length 120
+	$(PYTHON) -m flake8 tests detection_rules --ignore D203,N815 --max-line-length 120
 
 .PHONY: test
 test: $(VENV) lint pytest
 
+.PHONY: test-cli
+test-cli: $(VENV)
```

Comment:
I think it would be slightly cleaner to make this a bash script and call that with a single line - thoughts?

Reply:
Sounds good to me!